packages/expo-haptics/src/ExpoHaptics.web.ts

```diff
@@ -2,4 +2,7 @@ export default {
   get name(): string {
     return 'ExpoHaptics';
   },
+  async isAvailableAsync(): Promise<false> {
+    return false;
```

Comment:
And what about https://developer.mozilla.org/en-US/docs/Web/API/Vibration_API ? 🤔

Reply:
Yeah, I was playing with that as well. I actually started to implement this and worked pretty ok-ish. But since iOS doesn't support this, it might be misleading for people to say "its working on web". (https://caniuse.com/#search=vibration)